llpc/context/llpcCompiler.h

```diff
@@ -122,13 +122,16 @@ class Compiler : public ICompiler {
                                       ComputePipelineBuildOut *pipelineOut, void *pipelineDumpFile = nullptr);
   Result buildGraphicsPipelineInternal(GraphicsContext *graphicsContext,
                                        llvm::ArrayRef<const PipelineShaderInfo *> shaderInfo,
-                                       bool buildingRelocatableElf, ElfPackage *pipelineElf);
+                                       bool buildingRelocatableElf, ElfPackage *pipelineElf,
+                                       llvm::MutableArrayRef<CacheAccessInfo> stageCacheAccesses);
 
   Result buildComputePipelineInternal(ComputeContext *computeContext, const ComputePipelineBuildInfo *pipelineInfo,
-                                      bool buildingRelocatableElf, ElfPackage *pipelineElf);
+                                      bool buildingRelocatableElf, ElfPackage *pipelineElf,
+                                      CacheAccessInfo *stageCacheAccess);
```

Comment:
Consider taking `stageCacheAccess` by reference

Reply:
`ElfPackage *pipelineElf` uses a pointer for the output data.
Could you please explain why `stageCacheAccess` should be a reference?

Reply:
It makes it clear that there's a single non-null object behind it and there's no pointer arithmetic.